packages/dds/merge-tree/src/collections.ts

```diff
@@ -1079,11 +1107,11 @@ export class IntervalTree<T extends IInterval> implements IRBAugmentation<T, Aug
     }
 
     matchNode(node: IntervalNode<T> | undefined, key: T) {
-        return node && node.key.overlaps(key);
+        return !!node && node.key.overlaps(key);
     }
 
     continueSubtree(node: IntervalNode<T> | undefined, key: T) {
-        const cont = node && node.data.minmax.overlaps(key);
+        const cont = !!node && node.data.minmax.overlaps(key);
```

Comment:
The double-bang was new syntax for me. :)  https://medium.com/better-programming/javascript-bang-bang-i-shot-you-down-use-of-double-bangs-in-javascript-7c9d94446054